code/modules/jobs/job_types/captain.dm

```diff
@@ -33,6 +33,13 @@
 	species_outfits = list(
 		SPECIES_PLASMAMAN = /datum/outfit/plasmaman/command
 	)
+
+	minimal_lightup_areas = list(
+		/area/crew_quarters/heads/captain,
+		/area/crew_quarters/heads/hop,
```

Comment:
I added the HoP office here due to the fact the captain will often be doing hopline duty anyways if there's no HoP